config.ts

```diff
@@ -15,9 +15,15 @@ interface AuthConstants {
 }
 
 const devAuthConfig: AuthConfig = {
-    auth0Audience: "https://sandbox.api.material.com",
-    auth0ClientId: "ffh6DBHImNf7ZoTlL0sRmQU3EQ2iDuUx",
-    loginUrl: "https://materialsandbox.auth0.com/oauth/token",
+    auth0Audience:
+        process.env.ELEMENT_DEV_AUTH0_AUDIENCE ||
+        "https://sandbox.api.material.com",
+    auth0ClientId:
+        process.env.ELEMENT_DEV_AUTH0_CLIENT_ID ||
+        "ffh6DBHImNf7ZoTlL0sRmQU3EQ2iDuUx",
+    loginUrl:
+        process.env.ELEMENT_DEV_LOGIN_URL ||
+        "https://materialsandbox.auth0.com/oauth/token",
```

Comment:
I wonder if we should just keep the production configurations in the repo and control dev vs sandbox with the env variables.

Reply:
Yes, if we are saying 3rd party partners will be using production, then I can remove the sandbox settings and just have prod but with the env variable overrides.

Reply:
Yeah let's do it.